src/utils/clickHandler.js

```diff
@@ -55,6 +55,7 @@ module.exports = function clickHandler (e) {
     var isRedirectLink = isDefaultRedirectLink(target);
     var isPreventDefault = true;
     var props = self.props;
+    var followLink = props.followLink || props.follow;
```

Comment:
Need to check whether props.followLink exists, before falling back to props.follow.  i.e. if `props.followLink` is false, should just return false, instead of checking props.follow again.